main.go

```diff
@@ -30,3 +32,23 @@ func helloHandler(w http.ResponseWriter, r *http.Request) {
 	args := mux.Vars(r)
 	fmt.Fprintf(w, "Hello %s!", args["name"])
 }
+
+// initiateTranscriptionJobHandle takes a POST request containing a json object,
+// decodes it into an audioData struct, and returns the struct json-encoded.
+func initiateTranscriptionJobHandler(w http.ResponseWriter, r *http.Request) {
+	var jsonData transcriptionJobData
+
+	// unmarshal from the response body directly into our struct
+	if err := json.NewDecoder(r.Body).Decode(&jsonData); err != nil {
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+		return
+	}
+
+	// return the struct encoded as json
+	w.Header().Set("Content-Type", "application/json; charset=utf-8")
```

Comment:
How come you are returning the struct as json? Shouldn't we just return like "accepted"?